staff/jose-rodriguez/precourse/calculadora-html/main.js

```diff
@@ -0,0 +1,73 @@
+
+var num1 = 0;
+var num2 = 0;
+var op;
+
+
+function getNumber(numero) {
+    if(num1==0 && num1!=="0.") {
+        num1=numero;
+    } else {
+        num1 += numero;
+    }
+    showNum();
+}
+
+function dot() {
+    if (num1==0) {
+        num1 = '0,'
+    } else if (num1.indexOf(',') == -1) {
+        num1+= ','
```

Comment:
In lines 18 and 20 should substitute ',' for '.' , because it generates problems on the operations
2,3 + 1 != 2.3 =1

Reply:
done! thanks!!!!